app/src/test/java/org/simple/clinic/home/HomeScreenEffectHandlerTest.kt

```diff
@@ -56,4 +60,19 @@ class HomeScreenEffectHandlerTest {
         identifier = identifier
     ))
   }
+
+  @Test
+  fun `when open short code search screen effect is received, then open the screen`() {
+    // given
+    val shortCode = "123456"
+
+    // when
+    effectHandlerTestCase.dispatch(OpenShortCodeSearchScreen(shortCode))
+
+    // then
+    effectHandlerTestCase.assertNoOutgoingEvents()
+
+    verify(uiActions).openShortCodeSearchScreen(shortCode)
+    verifyNoMoreInteractions(uiActions)
```

Comment:
We should also be verifying no interactions in the other test, right?

Reply:
Good point missed it for one test. Fixing it